scripts/dotnet.py

```diff
@@ -39,6 +39,120 @@ def info(verbose: bool) -> None:
 ])
 
 
+class CompilationAction(Action):
+    '''
+    Tiered: (Default)
+
+    NoTiering: Tiering is disabled, but R2R code is not disabled.
+        COMPlus_TieredCompilation=0
+            This includes R2R code, useful for comparison against Tiered and
+            FullyJittedNoTiering for changes to R2R code or tiering.
+
+    FullyJittedNoTiering: Tiering and R2R are disabled.
+        COMPlus_TieredCompilation=0
+        COMPlus_ReadyToRun=0
```

Comment:
Include comment that this also sets `ZapDisable`

Reply:
Or if you don't want to duplicate the info maybe place the comments inline below where the env vars are set, then you don't have to mention which env vars are changed in the comments